### ISSUE TITLE
unit test error when run the file in test folder

### ISSUE THREAD

**Issue by 401qingkong** [microsoft/DeepSpeed#3044]
Hi， I install the deepspeed  package via the source code. And now I want to test the op in my platform. And I test with the following commands:
`cd tests;
pytest ./
`
But I find many import mistakes shows like below:
`/path/of/my/python/installed/python3.7/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/unit/inference/test_checkpoint_sharding.py:10: in <module>
    from transformers import AutoConfig, AutoModelForCausalLM
E   ModuleNotFoundError: No module named 'transformers'
`
I want to know how to run the test in the project. Hope for your replay

**Comment by tjruwase** [comment]
Have you installed `transformers`?

**Comment by 401qingkong** [comment]
> Have you installed `transformers`?

what's your transformers mean? I compile the project from source with the  args  DS_BUILD_TRANSFORMER=1. Do you mean I need to install the transformers package from pip?
`pip install transformers`

**Comment by tjruwase** [comment]
Yes, you need to do `pip install transformers`. Sorry, `DS_BUILD_TRANSFORMER=` is unrelated to transformers package.
